src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1882,7 +1884,7 @@ def client_side_proxy(cmd,
                       clientproxy_process=None):
 
     subscription_id = get_subscription_id(cmd.cli_ctx)
-
+    tenantId = _graph_client_factory(cmd.cli_ctx).config.tenant_id
```

Comment:
I think we made this change in earlier PR.

Reply:
We concluded that we won't make that change for proxy command right. Since there can be customers like backup team, who are using the additional values.yml  to set KAP tenantID. Due to this, KAP will contain user's tenantID and not resource's tenantID.

Reply:
Yes, didn't understand why this reflected as change. Are we reverting that changes in this PR?